app/models/course_data/courses_users.rb

```diff
@@ -117,6 +118,13 @@ def character_sum(revisions, namespace)
       .sum(:characters) || 0
   end
 
+  def refs_sum_ms(revisions)
```

Comment:
This method has the same name as the attribute, which I think will cause problems.

I also think we should avoid the shorthand "refs" and instead call the attribute something like `references_added`.